post-scan-actions/aws-python-promote-or-quarantine/README.md

```diff
@@ -308,3 +309,14 @@ To test that the Lambda function was deployed properly, you'll need to generate
 
     **NOTE:** It can take about 30-60 seconds or more for the file to move.
     </details>
+
+## Optional Configuration
+Additional options are supported as described in the following section.
+
+- **Skip promotion/quarantine**
+    - If you wish to _only_ promote or _only_ quarantine, remove the environment variable corresponding to the action you want to ignore. For example to skip quarantine and promote only, remove the `QUARANTINEBUCKET` environment variable from the scanner Lambda configuration.
+- **Custom ACL (Access Control List)**
+    - As stated in the [AWS documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl_overview.html) an ACL or access control list: "_...defines which AWS accounts or groups are granted access and the type of access_".
```

Comment:
- An access control list (ACL) [defines which AWS accounts or groups are granted access and the type of access](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl_overview.html) .

Reply:
Edit commited.